packages/moti/tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
   "extends": "expo-module-scripts/tsconfig.base",
   "compilerOptions": {
-    "jsx": "react-jsx",
     "outDir": "./build",
+    "customConditions": ["react-native-strict-api", "react-native"],
```

Comment:
This is what's used by Expo's base tsconfig, so I figured it'd be good to match it here.